packages/ui/package.json

```diff
@@ -43,9 +43,9 @@
   },
   "homepage": "https://github.com/draftbit/react-native-jigsaw#readme",
   "dependencies": {
-    "@draftbit/core": "48.2.2",
-    "@draftbit/native": "48.2.2",
-    "react-native-reanimated": "~2.12.0"
+    "@draftbit/core": "48.2.3",
+    "@draftbit/native": "48.2.3",
+    "react-native-reanimated": "~2.14.4"
```

Comment:
This was a version mismatch caused by an older commit. This is the proper version for expo 48.